optuna_dashboard/ts/components/GraphContour.tsx

```diff
@@ -278,9 +278,19 @@ const getAxisInfoForNumericalParams = (
   paramName: string,
   distribution: FloatDistribution | IntDistribution
 ): AxisInfo => {
-  const padding = (distribution.high - distribution.low) * PADDING_RATIO
-  const min = distribution.low - padding
-  const max = distribution.high + padding
+  let min = 0
+  let max = 0
+  if (distribution.log) {
+    const padding =
+      (Math.log10(distribution.high) - Math.log10(distribution.low)) *
+      PADDING_RATIO
+    min = Math.pow(10, Math.log10(distribution.low) - padding)
+    max = Math.pow(10, Math.log10(distribution.high) + padding)
+  } else {
+    const padding = (distribution.high - distribution.low) * PADDING_RATIO
+    min = distribution.low - padding
+    max = distribution.high + padding
+  }
```

Comment:
ref: to implement those lines, I refered to the optuna implementation. https://github.com/optuna/optuna/blob/master/optuna/visualization/_contour.py#L355-L360